src/language/parser.ts

```diff
@@ -1052,6 +1050,26 @@ export class Parser {
     });
   }
 
+  /**
+   * EnumValue : Name but not `true`, `false` or `null`
+   */
+  parseEnumValueName(): NameNode {
```

Comment:
renamed this to `parseEnumValueName()` because it's no longer parsing into an `EnumValue`. but still having it in its own function keeps things clean.